### ISSUE TITLE
Markdown preview window reduces in size when UI is zoomed

### ISSUE THREAD

**Issue by MikeDimmickMnetics** [Microsoft/vscode#23095]
- VSCode Version: Code 1.10.2 (8076a19fdcab7e1fc1707952d652f0bb6c6db331, 2017-03-08T14:02:52.799Z)
- OS Version: Windows_NT ia32 10.0.14393
- Extensions: 

|Extension|Author|Version|
|---|---|---|
|vscode-markdownlint|DavidAnson|0.7.0|
|vscode-eslint|dbaeumer|1.2.7|
|gitlens|eamodio|2.12.2|
|tslint|eg2|0.8.1|;

---

Steps to Reproduce:

1. Set zoom to 100% (e.g. press Ctrl+NumPad0)
2. Open a Markdown document in Preview mode (e.g. Release Notes)
3. Increase the zoom level by pressing Ctrl+=
4. Observe that a large blank area has appeared to the right and bottom of the preview
5. Increase the zoom level again by pressing Ctrl+=
6. Observe that the blank area has increased in size

The preview should occupy the whole client area of the tab, with no margin.

100% zoom:
![image](https://cloud.githubusercontent.com/assets/16961013/24248555/d62fd15e-0fc7-11e7-9e87-5cd1eb603da0.png)

After step 3:
![image](https://cloud.githubusercontent.com/assets/16961013/24248627/149db348-0fc8-11e7-95cb-8cd5f47c617d.png)

After step 5:
![image](https://cloud.githubusercontent.com/assets/16961013/24248650/23608d4c-0fc8-11e7-8333-02ee76f4ad3d.png)

**Comment by mjbvz** [comment]
Duplicate of #5745
